docs/intro.asciidoc

```diff
@@ -28,6 +28,20 @@ You can then use the APM app in Kibana to gain insight into latency issues and e
 
 More detailed information on how the Agent works can be found in the <<faq-how-does-it-work,FAQ>>.
 
+[float]
+[[experimental]]
+=== Experimental features
+
+Some agent features or <<supported-technologies-details,plugins>> are marked as `experimental`, which means:
+
+* They are opt-in (disabled by default).
+* There are no guarantees regarding changes in such features in future releases.
+* In some cases they have special caveats or unknowns that come with them.
+In such cases, the specific feature/plugin documentation should indicate that.
+
+There are cases where we decide to release a feature as `experimental` to be on the safe side and only make it an official feature after we get some feedback about it.
+Any such feedback will be highly appreciated, through a related GitHub issue or through the https://discuss.elastic.co/c/apm[APM discuss forum].
+
```

Comment:
I would remove this entire `Experimental features` section. I think it's repetitive. Here's why:

```
* They are opt-in (disabled by default).
* There are no guarantees regarding changes in such features in future releases.
```

The note below, included with all experimental features, already explains both of these points. 

```
NOTE: This feature is currently experimental, which means it is disabled by default and it is not guaranteed to be backwards compatible in future releases.
```

The third bullet point explains that documentation will indicate if there are caveats or unknowns.

The only thing missing is the feedback point. If that's important, I say we just update the experimental tag to something like this: 

```
NOTE: This feature is currently experimental, which means it is disabled by default and it is not guaranteed to be backwards compatible in future releases. Have feedback? Visit the https://discuss.elastic.co/c/apm[APM discuss forum].
```

Reply:
I agree. I added this because there are places that are not part of the configuration docs (like [the tuning page](http://apm-agent-java_1123.docs-preview.app.elstc.co/guide/en/apm/agent/java/master/tuning-and-overhead.html#circuit-breaker) or [supported technologies page](http://apm-agent-java_1123.docs-preview.app.elstc.co/guide/en/apm/agent/java/master/supported-technologies-details.html#supported-java-methods) where we just say `(experimental)`.
However, since in such places we DO refer to the enablement configuration, we can probably remove this section.

> The only thing missing is the feedback point

I don't think I want to add that into these configuration docs multiple times. It's ok to leave out.

Reply:
Sounds good to me. Thanks!